### PR TITLE
Update version and changelog for hackage-security/v0.6.2.1

### PR DESCRIPTION
**Pull request by Mikolaj**
New release. PR to make sure nobody objects and nobody wants to squeeze in a feature before the release.

### CODE REVIEW

File: hackage-security/ChangeLog.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 See also http://pvp.haskell.org/faq
 
+0.6.2.1
+-------
+
+* Allow GHC-9.0 (base-4.15)
+* Fix running `cabal repl hackage-security`
```
Comment (andreasabel) [inline hackage-security/ChangeLog.md:7]: This is #262 / #263, supposedly?

**Review by andreasabel — APPROVED** [review]

File: hackage-security/ChangeLog.md (hunk 1 above)
Comment (Mikolaj) [reply hackage-security/ChangeLog.md]: Right. Done.

**Comment by Mikolaj** [comment]
Candidate at https://hackage.haskell.org/package/hackage-security-0.6.2.1/candidate

**Comment by andreasabel** [comment]
@Mikolaj : I forgot to update the `tested-with` field in my PR. I should mention GHC 9.0.2, 8.10.7, 8.8.4.

**Comment by Mikolaj** [comment]
Let's update the field before the next release.

**Comment by Mikolaj** [comment]
Unless it bugs you. :)

**Comment by Mikolaj** [comment]
Published.

**Pull request merged**